frontend/lib/screens/settings_categories_screen.dart

```diff
@@ -45,7 +45,7 @@ class _CategoriesScreenState extends State<CategoriesScreen> {
             return Scaffold(
                 appBar: AppBar(
```

Comment:
The style of this app bar doesn't match those of the app bars in the rest of the app